2-mandatory/3-stocks.js

```diff
@@ -65,7 +89,21 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
-}
+    let arrayWithStrings = [];
+    for (let i = 0; i < stocks.length; i++) {
+        let stockOfName = stocks[i].toUpperCase();
+        let sortedArray = closingPricesForAllStocks[i].sort(function (a, b) {
```

Comment:
This looks good - I think it works.
There are a few other ways to do this as well - have a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max